update/renewCerts.bash

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+scriptDir="$(dirname $0)"
+cd $scriptDir
+# Read config file with paths to WP-installs and usernames
+source ../conf
+
+letsencFolder=/etc/letsencrypt
```

Comment:
I see your point but I hesitate to do something about it. You could put these constants in the wpManySiteAdmin config file but I think it adds unessecary clutter to that file.